Can't install via stack

with a `package.yaml` that looks like
```yaml
extra-deps:
- Hastodon-0.4.0
```
you get
```
Hastodon-0.4.0: configure
Hastodon-0.4.0: build
Progress: 1/2
--  While building custom Setup.hs for package Hastodon-0.4.0 using:
      /home/sam/.stack/setup-exe-cache/x86_64-linux-nopie/Cabal-simple_mPHDZzAJ_2.0.1.0_ghc-8.2.2 --builddir=.stack-work/dist/x86_64-linux-nopie/Cabal-2.0.1.0 build --ghc-options " -ddump-hi -ddump-to-file -fdiagnostics-color=always"
    Process exited with code: ExitFailure 1
    Logs have been written to: /home/sam/haskell-stuff/hast-test/.stack-work/logs/Hastodon-0.4.0.log

    Configuring Hastodon-0.4.0...
    Preprocessing library for Hastodon-0.4.0..
    Building library for Hastodon-0.4.0..
    [1 of 1] Compiling Web.Hastodon     ( Web/Hastodon.hs, .stack-work/dist/x86_64-linux-nopie/Cabal-2.0.1.0/build/Web/Hastodon.o )

    /tmp/stack6559/Hastodon-0.4.0/Web/Hastodon.hs:71:1: error:
        Could not find module ‘Web.Hastodon.API’
        Use -v to see a list of the files searched for.
       |
    71 | import Web.Hastodon.API
       | ^^^^^^^^^^^^^^^^^^^^^^^

    /tmp/stack6559/Hastodon-0.4.0/Web/Hastodon.hs:72:1: error:
        Could not find module ‘Web.Hastodon.Streaming’
        Use -v to see a list of the files searched for.
       |
    72 | import Web.Hastodon.Streaming
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    /tmp/stack6559/Hastodon-0.4.0/Web/Hastodon.hs:73:1: error:
        Could not find module ‘Web.Hastodon.Types’
        Use -v to see a list of the files searched for.
       |
    73 | import Web.Hastodon.Types
       | ^^^^^^^^^^^^^^^^^^^^^^^^^

    /tmp/stack6559/Hastodon-0.4.0/Web/Hastodon.hs:74:1: error:
        Could not find module ‘Web.Hastodon.Util’
        Use -v to see a list of the files searched for.
       |
    74 | import Web.Hastodon.Util
       | ^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/Hastodon.cabal
+++ b/Hastodon.cabal
@@ -35,4 +35,8 @@ Library
                  transformers
 
   Exposed-Modules: Web.Hastodon
-  Other-Modules:
+  Other-Modules: Web.Hastodon.API,
+                 Web.Hastodon.Streaming,
+                 Web.Hastodon.Types,
+                 Web.Hastodon.Util
+

